client/src/pages/css/PaymentPageOne.css

```diff
@@ -24,8 +24,10 @@ TODO: Style the progress bar to look more professional
     text-align: right;
 }
 
+.container {
```

Comment:
Was this the only change? The rest seem to be from the index.js PR. Let's merge that in first before checking out the diffs in this.

Reply:
Yeah it was